src/main/java/org/veupathdb/service/eda/ds/metadata/AppsMetadata.java

```diff
@@ -56,6 +58,9 @@ public class AppsMetadata {
           "A collection of visualizations designed to support the unbiased exploration of relationships between spatiotemporal variables in a cartographic map.",
           Arrays.asList(VECTORBASE_PROJECT),
           viz("map-markers", new StandaloneMapMarkersPlugin())),
+          viz("map-markers-overlay", new MapMarkersOverlayPlugin()),
```

Comment:
I had to remind myself what all the endpoints do.

`map-markers-overlay` is part of the legacy EDA visualizations for the "pass" app, and doesn't go here.